### ISSUE TITLE
map2

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#1286]
I'm pretty sure map2 is not involved in G2/M transition (way way upstream?)

what is the evidence?  should it be phenotype only?

https://curation.pombase.org/pombe/curs/5f9739dd2f7a875d/ro

**Comment by Antonialock** [comment]
they say

Thus, P-factor affects G2 cells in two ways: it reduces growth rate (Figure 8A) and it accelerates entry into mitosis (Figure 3A). A nutritional reduction of growth rate is known to cause mitosis at a smaller cell size (Fantes and Nurse, 1977), and so pheromone could have a similar effect.

**Comment by ValWood** [comment]
I'm sill confused by how this can work, but is this ti do with the penultimate  mitotic division  before  cells conjugate?

**Comment by Antonialock** [comment]
yeah. I don't mind removing it but I'm interested in how you think about that in terms of GO. Under the right conditions, binding of pheromone obviously sparks of the signaling cascade to go through G2/M at a smaller size, then arrest in G1 and search for a mating parner.

**Comment by ValWood** [comment]
I think we should  keep it for now but we need to be able to distinguish this from "canonical G2/M transition  regulation, I think, in the same way that we can distinguish
the nutritional starvation  part

GO:1905287
Name	positive regulation of G2/M transition of mitotic cell cycle involved in cellular response to nitrogen starvation

 
so something like
positive regulation of G2/M transition of mitotic cell cycle involved in cellular response to pheromone signalling

But we can sit on this for now, and address with the G2/M stuff....
